src/plots/geo/layout_attributes.js

```diff
@@ -174,9 +174,31 @@ var attrs = module.exports = overrideAll({
             min: 0,
             dflt: 1,
             description: [
-                'Zooms in or out on the map view.',
-                'A scale of *1* corresponds to the largest zoom level',
-                'that fits the map\'s lon and lat ranges. '
+                'Initial zoom level of the map view.',
+                'A scale of *1* (100%) corresponds to the base zoom level',
+                'that fits the map\'s lon and lat ranges.'
+            ].join(' ')
+        },
+        minscale: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
```

Comment:
Shouldn't `minscale` have a `max` range of `1`?

Reply:
I don't think so. Maybe it also make sense in some circumstances to only allow scaling between e.g. 2..3.

Than of course the initial scale should also set so one value between 2..3. This check is still missing! What would be a good place to check if `scale` is within the given min/max bounds and reset it otherwise? Is [layout_defaults.js](https://github.com/plotly/plotly.js/blob/76cb01f0789e151082612744f1d4218fb091234d/src/plots/geo/layout_defaults.js#L164-L166) the right place?

Reply:
Yes, layout_defaults is the right place - note that `coerce` returns the set value. Also we should check that `minscale <= maxscale`

Reply:
Also we probably need to check when the scale is automatically determined from the contents of the map whether it's within these bounds, and push it into the range if not. I'm not quite sure where that happens, possibly multiple places in geo.js depending on the situation.